Remove appenderMapStorageInitializer static variable, because:
(1) that's not how it should be declared; and (2) it's not used
internal to log4cpp & thus probably should not be externally either.
--- include/log4cpp/Appender.hh.orig	2017-04-18 00:50:54.000000000 -0500
+++ include/log4cpp/Appender.hh	2017-04-27 21:58:07.000000000 -0500
@@ -161,7 +161,6 @@
         static AppenderMapStorage &_appenderMapStorageInstance;
     };
 
-	static Appender::AppenderMapStorageInitializer appenderMapStorageInitializer; // static initializer for every translation unit
     typedef std::set<Appender *> AppenderSet;
 
 }
